Automattic/wp-calypso
# Upwork Banner: Migrate Illustration to Webpack

#### Changes proposed in this Pull Request

This migrates the two image uses of the Builder illustration to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=illustration-builder-referral.svg&unscoped_q=illustration-builder-referral.svg

Ensure that the two banners always display, for example by turning this to true: 

https://github.com/Automattic/wp-calypso/blob/a5180484034f2225d788d75190dd9c8166398f42/client/my-sites/stats/stats-banners/index.jsx#L107-L112

The banner should appear on the Stats page. Verify the image is identical.

<img width="1100" alt="Screenshot 2020-01-20 at 15 45 39" src="https://user-images.githubusercontent.com/43215253/72740125-f8016780-3b9c-11ea-8b82-af37d726ddd4.png">
